README.md

```diff
@@ -1,8 +1,46 @@
-# job_tracker
+# UpWiSE
 
-This is a WIP open source project to meet the following objectives:
+**UpWiSE** is an open-source project designed to help users efficiently track and organize their job applications. The app allows users to monitor their application progress, store notes, and attach relevant job links. It is created to simplify job hunting by consolidating all application details in one place.
 
-* Make an app for tracking job applications
-* Ensure that each entry can note the following: a job role name, a company name, a start date, an end date, a current status, and a way to create entries related to that application (saved related notes), and an optional link URL
+## Project Description
+
+The Job Tracker app allows users to:
+- Add and manage job applications with essential details such as job role, company, and dates.
+- Track the current status of each application (e.g., Applied, Interview, Offer).
+- Attach related notes to each job application.
+- Include an optional link for further reference (e.g., job listing URL).
+
+The aim of this project is to provide a user-friendly interface to streamline the job search process.
+
+## Technical Stack
```

Comment:
Once the initial project is checked in, this section and the initial setup section need to be updated.